Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/PrefabIntegrationManager.cpp

```diff
@@ -59,6 +60,7 @@ namespace AzToolsFramework
         PrefabFocusInterface* PrefabIntegrationManager::s_prefabFocusInterface = nullptr;
         PrefabFocusPublicInterface* PrefabIntegrationManager::s_prefabFocusPublicInterface = nullptr;
         PrefabLoaderInterface* PrefabIntegrationManager::s_prefabLoaderInterface = nullptr;
+        PrefabOverridePublicInterface* PrefabIntegrationManager::s_prefabOverridePublicInterface = nullptr;
```

Comment:
When are these static variables cleaned up?

Actually why are static variables being used here and not member variables of the PrefabIntegrationManager?

Reply:
> When are these static variables cleaned up?

I believe the the static variables get cleaned up when the application terminates. 

> Actually why are static variables being used here and not member variables of the PrefabIntegrationManager?

I've kept up with the existing pattern of declaring interface pointers static in the class. I'll try to make these non-static and see if anything breaks. I'm not sure if there is a particular reason any of these interfaces are declared static. Maybe @AMZN-daimini knows?

Reply:
I've made the interface to be a non-static member it works as expected. There are other interfaces in the class that are declared static as well that I'll discuss with @AMZN-daimini and modify in a later PR if needed

Reply:
I am trying to remember/dig out the reasoning behind why that pattern was established but I haven't really found a reason yet. I'll let you know if I figure it out.
Right now, though, I have no objections against turning them into member variables.